### PR TITLE
 Import OrderedSet class from ember.js

### PR DESCRIPTION
**Pull request by Turbo87**
/cc @locks @rwjblue

### CODE REVIEW

File: addon/index.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    if (presenceSet[guid] !== true) {
+      presenceSet[guid] = true;
+      this.size = list.push(obj);
+    }
+
+    return this;
+  }
+
+  /**
+    @since 1.8.0
```
Comment (rwjblue) [inline addon/index.js:60]: Seems like the `@since` docs should be removed

File: addon/index.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
+import { guidFor } from '@ember/object/internals';
+import { assert } from '@ember/debug';
+
+/**
+  This class is used internally by Ember and Ember Data.
+  Please do not use it at this time. We plan to clean it up
```
Comment (rwjblue) [inline addon/index.js:6]: Seems wrong?  Don't we actually want folks to feel free to use this?

File: tests/unit/ordered-set-test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { module, test } from 'qunit';
+
+import { isEmpty } from '@ember/utils';
+
+import OrderedSet from 'ember-ordered-set';
```
Comment (rwjblue) [inline tests/unit/ordered-set-test.js:5]: Are we going to use the scope?

**Review by rwjblue — APPROVED** [review]
Minor questions inline, feel free to land and iterate though...

File: addon/index.js (hunk 2 above)
Comment (Turbo87) [reply addon/index.js:6]: yes, I deliberately just imported the current state for now and will continue iterating on top of it

File: tests/unit/ordered-set-test.js (hunk 3 above)
Comment (Turbo87) [reply tests/unit/ordered-set-test.js:5]: good point. will address in a follow-up PR that renames the project.

File: addon/index.js (hunk 1 above)
Comment (locks) [reply addon/index.js:60]: Or corrected to 1.0.0 ;P

**Pull request merged**